src/style.css

```diff
@@ -0,0 +1,133 @@
+/* #region CSSreset */
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region Card styles */
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+.card__imagine {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__label {
+  margin: 0 16px;
+}
+
+.card__label p:first-child {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__label p:last-child {
+  margin: 4px 0 0;
+  margin-bottom: 9px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.card__feedback {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 16px;
```

Comment:
Can you add padding for the card? If you add padding, don’t need always add horizontal margins for blocks inside

Reply:
I just create variable ,because when added puddings it was not pixel perfect.